website/docs/intro/install/standalone.mdx

```diff
@@ -38,6 +38,10 @@ import Admonition from '@theme/Admonition';
 
 You can run OpenTofu without installation as a standalone binary. You can [download the latest release](https://github.com/opentofu/opentofu/releases/latest/) for your operating system from the [GitHub releases page](https://github.com/opentofu/opentofu/releases/latest/), unpack the zip and start using it. For easier updates, we recommend using the **non-portable packaged versions for your operating system**.
 
+### Community-supported Architectures
+
+Download the latest binary for [ppc64le here.](https://ftp2.osuosl.org/pub/ppc64el/opentofu/).
```

Comment:
I would recommend adding a disclaimer here to the effect that the OpenTofu project does not build or support these builds.

Additionally, I would recommend adding contact information on who is responsible for these builds and how to raise issues related to them.